pubspec.lock

```diff
@@ -458,5 +528,5 @@ packages:
     source: hosted
     version: "2.1.15"
 sdks:
-  dart: ">=2.1.1-dev.0.0 <3.0.0"
+  dart: ">=2.1.0 <3.0.0"
```

Comment:
use the `>= 2.2.0 < 3.0.0` as your SDK version. This will get you the latest version of mobx packages. You are still using the older versions without this change.

Reply:
Okay @pavanpodila